ROADMAP.md

```diff
@@ -13,79 +13,117 @@ for the issues tagged "area/X.Y.Z".
+
+Metadata
+
+   * Introduced support for artifact tracking via a UI and python SDK
+   * Predefined schemas for models, datasets, and evaluation metrics
+
+Multi-user support
 
-We are working diligently to get Kubeflow to its first major version release 1.0. Here's how we are approaching 1.0.
+   * ISTIO and RBAC for AuthZ
+   * Dex provides a reference implementation for AuthN
+   * Integration with GCP and AWS AuthN services
+   * Profile resource to manage per-team namespaces
+   * Kubeflow identity management API to manage multiple users/teams     
 
-* Individual applications within Kubeflow (e.g. TFJob, kfctl, Pipelines, etc...) will graduate to 1.0
-* We expect to start graduating some components in the second half of 2019
-* Once we have a set of graduated applications covering core CUJ's for Kubeflow we will graduate Kubeflow to 1.0
+Training
 
+   * TFJob and PyTorch to 1.0
 
-We are working on defining criterion for graduating applications. Here are some areas we think are important
+Pipelines
+   * UI improvements: 
+     * 10x Perf improvement (e.g. load list of runs) 
+   * API performance optimization
+   * streamlined run creation
+   * stackdriver integration for logs (better debugging)
+   * better visualization of metadata outputs
+   * Tensorboard CRD
+   * versioning for data volumes
 
 
-*   Stabilized APIs   
-*   Robust support for monitoring and logging.
-*   Scale and load testing.
+## Kubeflow 0.7
 
-## Enterprise Readiness
+Following Kubeflow's quarterly relese schedule Kubeflow 0.7 will be released in October 2019.
 
-The features in this enterprise readiness theme focus on better integration with existing enterprise infrastructure and support for secure data access. Some of the highlights in the area include:
+Notebooks
 
-*   Multi User Kubeflow Deployments
-	* [CUJ](http://bit.ly/kubeflow_cuj_multi_user)
-	* [Design Doc for Jupyter](http://bit.ly/kf_jupyter_design_doc)
-*   Isolation of environments within a cluster.
-*   RBAC and IAM integrations.
-*   Support for multi-tenancy.
-*   Hybrid/Multi-cluster deployments.
-*   Support for POSIX filesystems.
-*   [Issues](https://github.com/issues?utf8=%E2%9C%93&q=org%3Akubeflow+label%3Aarea%2Fenterprise_readiness+)
+  * Notebook infrastructure to beta quality in 0.7 
+    * 1.0 in the following release
+  * The notebook infrastructure has 2 primary pieces; the jupyter notebook controller
+    and web application for managing notebooks
+  * [Kanban board](https://github.com/orgs/kubeflow/projects/11)
 
-## Deployment Experience
+Deployment
 
-We have heard from our users and based on the feedback we are continuing to improve the deployment experience of Kubeflow. Here are some areas we are working on:
+   * kfctl to beta quality in 0.7
+     * 1.0 in the following release
+   * v1beta1 for KfDef
+     * Create a more consistent and clean API for describing Kubeflow deployments
+     * Handle plugins and platforms consistently
+   * Clean up kfctl flags and semantics
```

Comment:
@jlewi @abhi-g @ddutta 
A suggestion is we add 'accelerated training and inference support for deep learning models'.

We are prototyping this with an on-prem cluster so that a data scientist just runs a model 
and we have overlays in profile and tfoperator that injects device info that is optimal for that model.
We would also like to surface some of the device metrics up through the Application.

The value-add from matching a model (bert, MLP, transformer-lt) with hardware without requiring the data scientist know these details would be very compelling. This - for example - could be a profile overlay that could be prepackaged for common models or extended by vendors that are deploying kubeflow on known hardware and are focusing on specific models within their DS team.

Reply:
Do we have specific deliverables for the 0.7; 1.0 time frame that we should list here?

Reply:
yes
- adding device overlays in manifests/profile that can be included in the namespace and job operators